jetty-core/jetty-openid/src/test/java/org/eclipse/jetty/security/openid/OpenIdReamNameTest.java

```diff
@@ -11,15 +11,13 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.ee9.security.openid;
+package org.eclipse.jetty.security.openid;
```

Comment:
Please rename this file to `OpenIdRealmNameTest` (not the missing `l` in `Ream`).

Reply:
glad you pointed out the missing l.  I read it 10 times and thought it was the same :)